cmd/site-controller/controller.go

```diff
@@ -9,47 +9,47 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	corev1informer "k8s.io/client-go/informers/core/v1"
-	"k8s.io/client-go/informers/internalinterfaces"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
- 	"k8s.io/client-go/tools/cache"
+	corev1informer "k8s.io/client-go/informers/core/v1"
+	"k8s.io/client-go/informers/internalinterfaces"
+	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 
 	"github.com/skupperproject/skupper/api/types"
 	"github.com/skupperproject/skupper/client"
 )
 
 type SiteController struct {
-	vanClient     *client.VanClient
-	siteInformer  cache.SharedIndexInformer
-	tokenInformer cache.SharedIndexInformer
+	vanClient            *client.VanClient
+	siteInformer         cache.SharedIndexInformer
+	tokenInformer        cache.SharedIndexInformer
 	tokenRequestInformer cache.SharedIndexInformer
-	workqueue     workqueue.RateLimitingInterface
-	siteId        string
+	workqueue            workqueue.RateLimitingInterface
+	siteId               string
 }
 
 func NewSiteController(cli *client.VanClient) (*SiteController, error) {
-	siteInformer:= corev1informer.NewFilteredConfigMapInformer(
+	siteInformer := corev1informer.NewFilteredConfigMapInformer(
 		cli.KubeClient,
-		cli.Namespace,
+		metav1.NamespaceAll,
```

Comment:
If we had an env var that could specify the namespace, and would default to all if not present, that would let the site controller be used even without the need to grant it cluster level privileges.